transformer_lens/components.py

```diff
@@ -593,6 +600,27 @@ def forward(
             )
             / self.attn_scale
         )  # [batch, head_index, query_pos, key_pos]
+
+        # alibi encoding before applying causal mask
+        if self.cfg.positional_embedding_type == 'alibi':
+            #TODO: not sure about the side effect of not using standard, double check
```

Comment:
What does this mean?

Reply:
A reminder for myself to double check any potential side effect of setting embedding type to something other than `standard`, no longer needed, deleted!